src/config/index.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  apiBaseUrl: process.env.REACT_APP_BASE_URL,
```

Comment:
What value does "REACT_APP add to the name?

Reply:
Because the app was bootstrapped with `create-react-app`, every environment variable must be prefixed with `REACT_APP_`